lib/actions.js

```diff
@@ -104,6 +107,12 @@ actions.loadModels = function() {
     this.modelNames = results.map(function(m) {
       return m.name;
     });
+    this.editEnableModels = results.filter(function(result) {
```

Comment:
I find it difficult to understand the name `editEnableModel`, how about `editableModels` or `writeableModels`? Similarly for the list of model names below.